masterlist.yaml

```diff
@@ -9415,10 +9415,10 @@ plugins:
       - <<: *doNotClean
         condition: 'version("Requiem.esp", "1.9.3", >=)'
       - <<: *patchIncluded
-        condition: 'active("Unofficial Skyrim Patch.esp") and not active("Requiem - Bugsmasher Edition.esp")'
+        condition: 'version("Requiem.esp", "2.0.0", <) and active("Unofficial Skyrim Patch.esp") and not active("Requiem - Bugsmasher Edition.esp")'
```

Comment:
Should this version number not be `3.0.0`?

Reply:
Support for `Unofficial Skyrim Patch.esp` was dropped in 2.0.0.
So I think the condition is correct.
https://requiem.atlassian.net/wiki/spaces/RD/pages/403439801/Requiem+2.0.0+-+The+Phantom+Agony

Reply:
Oh, right.

Reply:
Yes, `Requiem - Bugsmasher Edition.esp` was removed in 2.0.0.